### ISSUE TITLE
Status parameter is actually State according to OSB docs

### ISSUE THREAD

**Issue by DorukAkinci** [meshcloud/unipipe-service-broker#79]
https://github.com/openservicebrokerapi/servicebroker/blob/v2.14/spec.md#body-1

I have seen that we use wrong parameter name for `status` and we should change them as `state` if we can. We don't comply with Open service broker documents at all when it comes to this param. Is this a typo issue or do we have any purpose around it?

**Comment by JohannesRudolph** [comment]
Good catch. I assume it was an oversight when implementing the original OSB model in the java part of the broker. 
I would argue that we should try keeping the models that unipipe uses as close to the OSB spec as possible (principle of least astonishment, lessens our documentation burden since we can just point to the spec). 

However, unipipe does not exactly follow 100% OSB API model in other places as well, e.g. unipipe's `ServiceInstance` representation includes the additional `deleted` field, which is clearly not inside the OSB API.

I guess fixing this rename is going to be hard to change considering that renaming this crucial part of the model will break many existing unipipe pipelines/workflows. We should keep fixing this naming issue it in mind for a 2.0 release of unipipe though.
